src/ui/pages/SendForm/SendForm.tsx

```diff
@@ -433,6 +435,40 @@ function SendFormComponent() {
                 networkId={formState.tokenChain || null}
               />
             ) : null}
+            {preferences?.configurableTransactionData &&
+            addressType === 'evm' ? (
+              <FormFieldset
+                title="Data"
+                style={{ borderBottomRightRadius: 4 }}
+                startInput={
+                  <textarea
```

Comment:
Do we need an UnstyledTextarea element?

Reply:
Probably! Once it's used somewhere else perhaps....